enclone_core/src/defs.rs

```diff
@@ -249,6 +249,32 @@ pub struct GeneralOpt {
     pub session_narrative: String,
 }
 
+impl GeneralOpt {
+    pub fn is_bcr(&self) -> bool {
+        if self.tcr {
+            assert!(!self.bcr);
+            return false;
+        }
+        // The original logic for computing this value was based on the confusing
+        // assumption that this is always true if self.tcr is false.
+        // I've preserved this for now, but this line might make sense to add:
+        // assert!(self.bcr);
```

Comment:
Makes sense! We could use enum instead of two booleans

Reply:
Yeah, the logic here was just so odd/implicit that I decided to back away slowly before turning and running